tutorials/34_Model_Based_Evaluation_of_RAG_Pipelines.ipynb

```diff
@@ -0,0 +1,4480 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    pip install haystack-ai
</pre><p>Do we need to install haystack-ai since we install <code>uptrain-haystack</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepset-ai/haystack-tutorials/pull/283/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
No we don't but this way it's more visible to the user. And when `pip install uptrain-haystack` is executed it will notice that `haystack-ai` is already installed and just skip that substep.